lib/services/role.go

```diff
@@ -91,6 +91,15 @@ func RoleNameForCertAuthority(name string) string {
 // NewAdminRole is the default admin role for all local users if another role
 // is not explicitly assigned (this role applies to all users in OSS version).
 func NewAdminRole() Role {
+	// DELETE IN: 5.1.0
+	//
+	// Only needed until 5.1 when user and token management will be added to OSS.
+	adminRules := CopyRulesSlice(AdminUserRules)
+	if modules.GetModules().ExtendAdminUserRules() {
+		adminRules = append(adminRules,
+			NewRule(KindUser, RW()), NewRule(KindToken, RW()))
+	}
```

Comment:
Note for reviewers. The reason additional rules are being added here instead of simply returning a different set of rules is due to a circular imports issue that the package `modules` and `services` get into. To break the circular import, I would have to move all the `Kind*` constants and `R{O,W}` functions out and convert `services.Rule` to an interface.

Didn't feel the extra work was worth the effort for something that will be deleted in a few weeks.

Reply:
You could have an additional set of rules with the comment to delete in 5.1 and avoid the append:
```go
adminRules := CopyRulesSlice(AdminUserRules)
if modules.GetModules().ExtendAdminUserRules() {
	adminRules = CopyRulesSlice(AdminExtUserRules)
}
```

Reply:
Fixed.